### ISSUE TITLE
Visualiser widget crashing

### ISSUE THREAD

**Issue by Aneeqasif** [elParaguayo/qtile-extras#119]
whenever i restart qtile while cava is running wm just crash and i fell onto sddm my login manager

**Comment by elParaguayo** [comment]
Thanks for the report and sorry to hear about the crash. 

How are you restarting qtile?

**Comment by Aneeqasif** [comment]
```python
    Key([mod, "control"], "r", lazy.reload_config(), desc="Reload the config"),
```

i am using keybinding to reload.
this issue is also there if i do
```qtile cmd-obj -o cmd -f reload_config```

**Comment by elParaguayo** [comment]
Can't recreate this. Reloading the config works correctly for me.

**Comment by elParaguayo** [comment]
Thanks for the update. I'll need to test some more but won't have a chance until next week.

**Comment by elParaguayo** [comment]
I'm still not able to recreate this. 

@Aneeqasif @gibranlp are you still getting crashes on the latest version (NB you can only use latest version if you're also using the git version of qtile).

**Comment by elParaguayo** [comment]
Ok. I get an occasional crash when I `stop()` the widget, Interesting...

**Comment by elParaguayo** [comment]
This should hopefully be fixed in #158. Please test if you can.
